src/ContainerDynamicReturnTypeExtension.php

```diff
@@ -28,16 +28,16 @@ public function isMethodSupported(MethodReflection $reflection): bool
 
     public function getTypeFromMethodCall(MethodReflection $reflection, MethodCall $methodCall, Scope $scope): Type
     {
-        if (count($methodCall->args) === 0) {
+        if (count($methodCall->getArgs()) === 0) {
```

Comment:
Better call `getArgs()` only once and save the result in a variable?

Reply:
yup, done.